### PR TITLE
Implemented support to search Job on Pipeline page(#2adk6nd)

### PR DESCRIPTION
**Pull request by rathoreprashant**
Added search functionality to search jobs by job description 
**Please Review**
Before
Before
After
after

### CODE REVIEW

File: src/views/Pipeline.vue [hunk 1]
```diff
@@ -355,6 +359,7 @@ export default defineComponent({
       return DateTime.local().plus(timeDiff).toRelative();
     },
     async loadMoreJobHistory(event: any){
+      if(this.queryString.length>0) this.findJobs();
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:362]: I think this do an extra API call for get initial records for all the segments

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Pipeline.vue [hunk 2]
```diff
@@ -381,27 +385,29 @@ export default defineComponent({
     async refreshJobs(event: any) {
       this.isRetrying = true;
       if(this.segmentSelected === 'pending') {
-        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0}).then(() => {
+        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0,queryString:this.queryString}).then(() => {
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:388]: Spacing issue

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/views/Pipeline.vue (hunk 2 above)
Comment (rathoreprashant) [reply src/views/Pipeline.vue]: Fixed, Please have a look sir

File: src/views/Pipeline.vue [hunk 3]
```diff
@@ -22,6 +22,7 @@
     </ion-header>
 
     <ion-content>
+      <ion-input :placeholder="$t('Search jobs')" v-model="queryString" v-on:keyup.enter="segmentSelected === 'pending' ? getPendingJobs() :segmentSelected === 'running' ? getRunningJobs():getJobHistory()"/>
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:25]:
```suggestion
      <ion-input :placeholder="$t('Search jobs')" v-model="queryString" v-on:keyup.enter="segmentSelected === 'pending' ? getPendingJobs() : ( segmentSelected === 'running' ? getRunningJobs() : getJobHistory())"/>
```

File: src/views/Pipeline.vue [hunk 4]
```diff
@@ -201,7 +202,7 @@
 
           <ion-refresher slot="fixed" @ionRefresh="refreshJobs($event)">
             <ion-refresher-content pullingIcon="crescent" refreshingSpinner="crescent" />
-          </ion-refresher>
+          </ion-refresher>
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:205]:
```suggestion
          </ion-refresher>
```

File: src/views/Pipeline.vue [hunk 5]
```diff
@@ -381,27 +385,29 @@ export default defineComponent({
     async refreshJobs(event: any) {
       this.isRetrying = true;
       if(this.segmentSelected === 'pending') {
-        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0}).then(() => {
+        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0, queryString:this.queryString}).then(() => {
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:388]:
```suggestion
        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0, queryString: this.queryString}).then(() => {
```

File: src/views/Pipeline.vue [hunk 6]
```diff
@@ -381,27 +385,29 @@ export default defineComponent({
...
-        this.store.dispatch('job/fetchJobHistory', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0}).then(() => {
+        this.store.dispatch('job/fetchJobHistory', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0, queryString:this.queryString}).then(() => {
           if(event) event.target.complete();
           this.isRetrying = false;
         });
       }
     },
+
     segmentChanged (e: CustomEvent) {
+      this.queryString="";
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:406]:
```suggestion
      this.queryString = "";
```

**Review by adityasharma7 — CHANGES_REQUESTED** [review]

File: src/store/modules/job/actions.ts [hunk 7]
```diff
@@ -37,7 +37,7 @@ const actions: ActionTree<JobState, RootState> = {
     }
     return resp;
   },
-
+
```
Comment (adityasharma7) [inline src/store/modules/job/actions.ts:40]:
```suggestion

```

File: src/views/Pipeline.vue (hunk 3 above)
Comment (adityasharma7) [inline src/views/Pipeline.vue:25]: Add Search jobs to en.json

File: src/views/Pipeline.vue [hunk 8]
```diff
@@ -381,27 +385,29 @@ export default defineComponent({
...
       this.isRetrying = true;
       if(this.segmentSelected === 'pending') {
-        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0}).then(() => {
+        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0, queryString: this.queryString}).then(() => {
           if(event) event.target.complete();
           this.isRetrying = false;
         });
       } else if(this.segmentSelected === 'running') {
-        this.store.dispatch('job/fetchRunningJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0}).then(() => {
+        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0, queryString: this.queryString}).then(() => {
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:393]:
```suggestion
        this.store.dispatch('job/fetchRunningJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0, queryString: this.queryString}).then(() => {
```

File: src/views/Pipeline.vue [hunk 9]
```diff
@@ -381,27 +385,29 @@ export default defineComponent({
...
           if(event) event.target.complete();
           this.isRetrying = false;
         });
       } else {
-        this.store.dispatch('job/fetchJobHistory', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0}).then(() => {
+        this.store.dispatch('job/fetchPendingJobs', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0, queryString: this.queryString}).then(() => {
```
Comment (adityasharma7) [inline src/views/Pipeline.vue:398]:
```suggestion
        this.store.dispatch('job/fetchJobHistory', {eComStoreId: this.getCurrentEComStore.productStoreId, viewSize:process.env.VUE_APP_VIEW_SIZE, viewIndex:0, queryString: this.queryString}).then(() => {
```

**Review by adityasharma7 — APPROVED** [review]

**Pull request merged**